andela/ah-shakas-frontend
# #160577455 bookmark articles

## What does this PR do?
- It implements functionality for bookmarking articles to be read later.

## Description of Task to be completed?
- Users should be able to bookmark articles for reading later via a bookmark tab, button or link.

## How should this be manually tested?
- Clone this branch ft-bookmark-160577455
- Run the following commands on your terminal:
    `yarn install` - to install required dependencies
    `yarn start` - to start the server
- When the server starts, you will be redirected to the browser, where you can view articles.
- To bookmark an article, signup then login.
- After successful login, click on an article, and notice the bookmark icon to the far right.
- Click on it to bookmark, and click on it again in case you want to un-bookmark the article.
**Screenshot**
![image](https://user-images.githubusercontent.com/34505753/48856867-955de180-edc8-11e8-832b-12caa1ff6c2b.png)


## Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?
[#160577455](https://www.pivotaltracker.com/n/projects/2198533/stories/160577455)

## Deployment note
N/A

## Related PRs branch
[ft-bookmark-160577455](https://github.com/andela/ah-shakas-frontend/tree/ft-bookmark-160577455)

## Todos
- [x ] Test
